docs/api/java-api-quickstart.md

```diff
@@ -66,6 +66,17 @@ Table table = catalog.createTable(name, schema, spec);
 
 The logs [schema](#create-a-schema) and [partition spec](#create-a-partition-spec) are created below.
 
+HiveCatalog supports registering the table using a metadata location as below:
+
+```java
+import org.apache.iceberg.Table;
+import org.apache.iceberg.catalog.TableIdentifier;
+
+TableIdentifier name = TableIdentifier.of("logging", "logs");
+// register an existing table to this HiveCatalog.
+Table table = catalog.registerTable(name, metadata_file_location);
```

Comment:
+1 on documenting this, do we need to additionally call-out that this needs to be a json file and compliant with metadata.json format ? as per [CP](https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/TableMetadataParser.java#L261-L268)

or is it self implied, Your thoughts ?

Reply:
I left it out thinking it is implicit. 
Updated it now.